modules/management-console-agent/src/test/java/org/apache/ignite/agent/AgentCommonAbstractTest.java

```diff
@@ -93,16 +105,50 @@ public void stopAndClear() throws Exception {
     /**
      * @param ignite Ignite.
      */
-    protected void changeManagementConsoleUri(IgniteEx ignite) {
+    protected void changeManagementConsoleConfig(IgniteEx ignite) {
+        changeManagementConsoleConfig(ignite, true);
+    }
+
+    /**
+     * @param ignite Ignite.
+     */
+    protected void changeManagementConsoleConfig(IgniteEx ignite, boolean isAssertNeeded) {
+        boolean isSslEnabled = stream(environment.getActiveProfiles()).anyMatch("ssl"::equalsIgnoreCase);
+        boolean isProxyEnabled = getBoolean("test.withProxy");
+
         ManagementConfiguration cfg = ignite.context().managementConsole().configuration();
 
-        cfg.setConsoleUris(F.asList("http://localhost:" + port));
+        if (isSslEnabled) {
+            boolean isKeyStoreNeeded = getBoolean("test.withKeyStore");
+            boolean isTrustStoreNeeded = getBoolean("test.withTrustStore");
+
+            if (isTrustStoreNeeded) {
+                cfg.setConsoleTrustStore(AgentCommonAbstractTest.class.getClassLoader().getResource("ssl/server.p12").getPath());
+                cfg.setConsoleTrustStorePassword("123456");
+            }
+
+            if (isKeyStoreNeeded) {
+                cfg.setConsoleKeyStore(AgentCommonAbstractTest.class.getClassLoader().getResource("ssl/client.p12").getPath());
+                cfg.setConsoleKeyStorePassword("123456");
+            }
+        }
+
+        if (!isProxyEnabled)
+            cfg.setConsoleUris(F.asList((isSslEnabled ? "https" : "http") + "://localhost:" + port));
+        else {
```

Comment:
we do not need `{}` here, just use `else if (...)`

Reply:
But we have another else after **if (isWindows() || isMac())**  :(